src/linux/inc/lxwil.h

```diff
@@ -373,6 +373,11 @@ class unique_fd
         return fd;
     }
 
+    int* addressof() noexcept
```

Comment:
Should we override the address-of operator instead?

Reply:
Potentially. This felt a bit easier to read (similar to the put()) method that wil has